### DISCUSSION TITLE
Have problem detect single digit

### DISCUSSION THREAD

**Discussion by patrickjchen** [mindee/doctr#1265] [General]
Hi, thanks for the package.
However I am having problem with the attached image. Seems doctr can not detect single digit.
Any special config to use to make it work?
Thanks!
Patrick

![img1](https://github.com/mindee/doctr/assets/20728658/b96e3146-3994-4326-8f24-a0586d90f196)

**Comment by nikokks** [comment]
Hi,

Maybe the models have been trained like that: detect at least 2 charset and return a box .
By running the demo app with default models, it seems that the detection model doesn't create boxes.
You can see it as follows:
![Screenshot from 2023-05-30 12-17-01](https://github.com/mindee/doctr/assets/40694487/60bed700-ca7b-4fe1-82b2-5f0a4a12ce20)

I think with better SOTA models it would detect it.
In next doctr versions, contributors will implement them

**Comment by patrickjchen** [comment]
Hi Nocholas,
Thanks for the reply. When will the next version be out? In the mean time,  is there a way for me to train it myself without too much effort? 

Thanks
Patrick

**Comment by nikokks** [comment]
Hello @patrickjchen,

Thank you for sharing your thoughts and ideas. The fastest way to achieve your desired results seems to train yourself on one of the models of detection with tagged data, specifically focusing on texts with single character/digit labels. It could be an effective solution.. but not in the long term.

I'd be more than happy to assist you by providing relevant data that aligns with your specific interests. By fine-tuning the data you choose, you can further optimize your progress. Moreover, if you decide to share the data you've tagged with doctr, it would not only benefit in the future yourself but also other people. It would greatly contribute to enhance future models and ensuring they incorporate your desired specifications.

Would you be interested ? :)

**Comment by patrickjchen** [comment]
Thanks Nikokks!   However,  ML/AI is just a hobby for me. I will contact you in a couple of weeks when I have some more free time.

Best Regards
Patrick
